kustomize/install/singlenamespace/manager-target.yaml

```
@@ -0,0 +1,13 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pgo
+spec:
+  template:
+    spec:
+      containers:
+      - name: operator
+        env:
+        - name: PGO_TARGET_NAMESPACE
+          valueFrom: { fieldRef: { apiVersion: v1, fieldPath: metadata.namespace } }
```

Why express like this vs
```suggestion
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: metadata.namespace
```

---

This is a habit I picked up when there are lots of variables.

```yaml
- name: a
  value: 'x'
- name: b
  valueFrom: { ... }
- name: c
  value: 'y'
```

Why spread it over more than one line?